src/request.ts

```diff
@@ -20,6 +20,8 @@
 import axios, { AxiosInstance, AxiosRequestConfig } from 'axios';
 import fs from 'fs';
 import { HttpProxyAgent, HttpsProxyAgent } from 'hpagent';
+import https from 'https';
+import forge from 'node-forge';
```

Comment:
do we want to add another dependency over the old certificate solution?

https://github.com/SonarSource/sonar-scanner-npm/blob/master/src/config.js#L171

what are the benefits here?

Reply:
Unfortunately, the old certificate resolution doesn't support the truststore (PEM vs PKCS12), and the PKCS12 truststore is a requriement for Java, so the idea is to also use it for Node

In the event where the user needs a workaround (because our implementation is weaker than java's impl to read the truststore), he can use `NODE_EXTRA_CA_CERTS` to pass additional cas to Node. We can also assess there and implement a more robust solution.